### PR TITLE
Place crc binary at the root of the tray

### PR DESCRIPTION
**Pull request by gbraad**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -25,18 +25,33 @@ const delay = ms => new Promise(resolve => setTimeout(resolve, ms))
...
+
 function crcBinary() {
   if (app.isPackaged) {
-    return path.join(app.getAppPath(), 'crc');
+    if (isWin) {
+      return path.join(app.getAppPath(), '..', '..', 'crc');
+    }
+    if (isMac) {
+      return path.join(app.getAppPath(), '..', 'crc');
+    }
```
Comment (praveenkumar) [inline main.js]: @gbraad Are we sure this is right thing to do, using `..` here?

Comment (gbraad) [reply main.js]: For Windows this works to navigate to `C:\Program Files\CodeReady Containers`.

Comment (gbraad) [reply main.js]: image

Comment (gbraad) [reply main.js]: @praveenkumar not sure what to do for macOS.

I can remove the if statement for macOS and as default return the `appPath`. WDYT?

**Pull request merged**